core/web/middleware.go

```diff
@@ -1,18 +1,36 @@
 package web
 
 import (
+	"embed"
 	"fmt"
+	"io/fs"
 	"net/http"
 	"os"
 	"path"
 	"strconv"
 	"strings"
 
 	"github.com/gin-gonic/gin"
-	"github.com/gobuffalo/packr"
+
 	"github.com/smartcontractkit/chainlink/core/logger"
 )
 
+// Go's new embed feature doesn't allow us to embed things outside of the current module.
+// To get around this, we basically need to copy over the assets to this module's directory
```

Comment:
You could try to output the yarn into the module instead perhaps?

Reply:
Or a symlink could work?

Reply:
I believe you cannot use symlinks for security reasons

Reply:
If we output it to `core/web`, will that break things like running the frontend server separately? Or will that end up re-compiling assets anyway (without minifying probably, to aid in debugging)?